dev/helpers/mssql/.gitignore

```diff
@@ -0,0 +1 @@
+all_migrations.txt
```

Comment:
💭 How about just cleaning up, or generating a temp file e.g. `tempfile` instead of this?

Reply:
This is done

Reply:
I could have been more clear here. What you have now works, but I meant the literal `tempfile` command that happens to be OS-specific but more generally I am thinking about `mktemp`: https://www.gnu.org/software/autogen/mktemp.html -- this is usually the approach I've seen.